app/main.py

```diff
@@ -1 +1,60 @@
 #  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True and self.appetite != 0:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
```

Comment:
And you don't need to use an `else`  condition, because if the previous condition is False, you just need to return 0. You can remove the `else`  condition and shift `return 0` 4 spaces to the left